resources/js/components/SignIn.js

```diff
@@ -0,0 +1,120 @@
+import * as React from 'react';
+import { Link as RouterLink } from 'react-router-dom';
+import Footer from './layouts/Footer';
```

Comment:
you can do the MUI imports first before custom components

```
// imports for react related packages
----- line space -----
// imports for MUI
----- line space -----
// imports for custom project components
```

Reply:
I have organized my imports na ma'am